miden-tx/tests/test_miden_faucet_contract.rs

```diff
@@ -246,6 +265,7 @@ fn test_faucet_contract_creation() {
 }
 
 fn get_faucet_account_with_max_supply_and_total_issuance(
+    public_key: PublicKey,
```

Comment:
If we implement the change for key pair creation then this can be changed to `public_key: Word`